mcs/tools/wasm-tuner/InterpToNativeGenerator.cs

```diff
@@ -171,6 +171,9 @@ class InterpToNativeGenerator {
 		"VIIIFII",
 		"VIIIFIII",
 		"VIIIIF",
+		"IFFFFIII",
+		"IFFIII",
+		"VIIIIFFII",
```

Comment:
please also add the signatures I added in https://github.com/dotnet/runtime/commit/aa667de1e3fff0f951d5136a45fbe8acefad9be9#diff-997ede58ca6a9458c6045aca1006b2faR1658, didn't have time to backport those to mono/mono yet.

Reply:
Done.